src/event.rs

```diff
@@ -1,4 +1,23 @@
-//! This module contains the `WindowEvent` type and re-exports glutin event
+//! This module contains the `Event` enum and re-exports glutin event
 //! types.
 
-pub use winit::{ElementState, Event, KeyboardInput, MouseButton, VirtualKeyCode, WindowEvent};
+pub use winit::{DeviceEvent, DeviceId, ElementState, KeyboardInput, MouseButton, VirtualKeyCode, WindowEvent};
+
+/// Events that the engine might send to the user.
```

Comment:
Do we want to either give a bit more detail on what is within `WindowEvent` or link to the winit crate? Might make it easier for newer people to figure out how to use the input system.

Reply:
Input system is managed automatically by the amethyst crate, so users aren't required to pump events themselves.  However as @torkleyy said I think I'm going to flatten this hierarchy anyways.